purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -674,6 +678,7 @@ internal class BillingWrapper(
                     stateListener?.onConnected()
                     executePendingRequests()
                     reconnectMilliseconds = RECONNECT_TIMER_START_MILLISECONDS
+                    trackFeatureNotSupportedIfNeeded()
```

Comment:
We should add some tests here. Not sure if you were waiting for a review though.